RoundStartFreezeTimeManager.cs

```diff
@@ -64,6 +67,17 @@ public void Start()
                 _countdownTimer = null;
             }
 
+            if (_soundTimer != null)
+            {
+                _soundTimer?.Kill();
+                _soundTimer = null;
+            }
+
+            if (EnableCountDownSound == true)
+            {
+                _soundTimer = _plugin.AddTimer(1.0f, PlaySound, TimerFlags.REPEAT);
```

Comment:
There's a bit of an issue with this, though pretty obscure and probably never going to be noticed by anybody. The amount of freezetime seconds is a float, meaning it can be e.g. .5 seconds. As this timer is started when freezetime starts, in that scenario it will play the sound offset .5 seconds into the second.

It's not that big of a deal, though.

Reply:
im not sure how i would aproach it different if you have any idea how it could be better to do it please give me some feedback, i am kinda a beginner. i have learned everything i know with just scrolling through plugins and try to understand what they do an how they work :D

Reply:
Yeah it's not a massive problem. I'll likely redo this class at some point since it's a bit cursed